src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,130 @@
+.menu {
+  display: none;
+  position: fixed;
+  inset: 0;
+  height: 100vh;
+  background-color: #d12d35;
+  padding: 0;
+
+  transition: all 2s ease-out;
+
+  opacity: 0;
+  pointer-events: none;
+  overflow: auto;
+
+  &:target {
+    display: block;
+    opacity: 1;
+    pointer-events: all;
+  }
+
+  &__bar {
+    padding: 25px 20px 129px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    z-index: 2;
+
+    @include on-tablet {
+      padding: 50px 72px 0;
+    }
+
+    @include on-desktop {
+      padding: 50px 140px 83px 120px;
+    }
+
+    &-logo {
+      &-image {
+        width: 64px;
+        height: 46px;
+
+        @include on-tablet {
+          width: 87px;
+          height: 65px;
+        }
+      }
+    }
+  }
+
+  &__icons {
+    display: flex;
+    gap: 24px;
+
+    @include on-tablet {
+      gap: 48px;
+    }
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+    font-size: 22px;
+    font-weight: 700;
+    padding: 0 20px 32px;
+    text-transform: uppercase;
+
+    @include on-tablet {
+      padding: 0 72px 32px;
+    }
+
+    @include on-desktop {
+      padding: 0 120px 48px;
+    }
+  }
+
+  &__container > a {
+    width: fit-content;
+    color: #fff;
+    text-decoration: none;
+    letter-spacing: 2px;
+  }
+
+  &__contacts {
+    padding: 0 20px;
+
+    @include on-tablet {
+      padding: 0 72px;
+    }
+
+    @include on-desktop {
+      padding: 0 120px 16px;
+    }
+  }
+
+  &__contacts--address {
+    display: flex;
+    flex-direction: column;
+    font-size: 16px;
+    font-weight: 700;
+    padding: 0 0 8px;
+    color: #fff;
+    text-transform: uppercase;
+
+    & > a {
+      display: block;
+      color: #fff;
+      text-decoration: none;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 27px;
+      width: fit-content;
+    }
+
+    &--call-to-order {
+      padding-top: 8px;
+      color: #fff;
+      font-size: 12px;
+      font-weight: 700;
+      text-transform: uppercase;
+      letter-spacing: 2px;
+      text-decoration: none;
+
+      &:hover {
+        cursor: pointer;
+        border-bottom: 1px solid #fff;
+        padding-bottom: 7px;
+      }
```

Comment:
Consider adding this style to every link in your menu for improved usability, it is always good to clearly indicate (through highlights and cursor changes) that an element is clickable [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Hightlight%20clickable%20element%20with%20cursor%3A%20pointer%20and%20some%20%3Ahover%20styles):

Your site:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/fca05e80-1fa3-4600-a066-fde0f1634c3d)

Figma:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/1b8fa18e-7969-4d26-aed9-4bfe84eca04a)